src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: inline-block;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.active {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
You should add a modifier to stars__star, not a new block
Checklist 2nd point

Reply:
I don't understand what i need to do here(
How can i get different stars without creating a new block

Reply:
you can use pseudo classes like nth-child